tests/pending/run/t8199.scala

```diff
@@ -39,8 +39,10 @@ object Test extends dotty.runtime.LegacyApp {
     checkClassName(c.getName)
   }
   def checkClassName(name: String): Unit = {
-    val defaultMaxClassFileLength = 255
-    assert((name + ".class").length <= defaultMaxClassFileLength, name)
+    final val MaxSuffixLength = 7 // "$.class".length + 1 // potential module class suffix and file extension
```

Comment:
Let’s drop the “+1” from the comment it seems an odd typo in the original PR

Reply:
".class" comes up to 6 characters so +1 explains 7 as the max suffix length. What do you mean by typo?

Reply:
but it says `$.class` not `.class`

Reply:
I assumed $ referred to the name of the class. Does the full suffix ever include `$`?

Reply:
Yes, $ is the module class suffix. 
https://github.com/scala/scala/pull/7497#discussion_r239070873 already mentioned some confusion on this comment...

Reply:
I see. Just pushed commit ammend